src/overlays/actors/ovl_Boss_Ganon2/z_boss_ganon2.c

```diff
@@ -2612,7 +2612,7 @@ void func_80904D88(BossGanon2* this, GlobalContext* globalCtx) {
         }
         gSPDisplayList(POLY_XLU_DISP++, ovl_Boss_Ganon2_DL_00B308);
 
-        for (i = 0; i < 15; i++) {
+        for (i = 0; i < ARRAY_COUNT(this->unk_234) - 1; i++) {
```

Comment:
I thought this looked weird, so I took a look at all instances where unk_234 is used within z_boss_ganon2. It appears as if this array should only be 15 elements long and not 16, as the limb index lookup tables only use 0x0E as a max index.

Reply:
I think you are right. I've now changed the length.